frontend/src/components/images/imageBox.jsx

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import clsx from 'clsx';
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import CardActions from '@material-ui/core/CardActions';
+import Collapse from '@material-ui/core/Collapse';
+import IconButton from '@material-ui/core/IconButton';
+import Typography from '@material-ui/core/Typography';
+import FavoriteIcon from '@material-ui/icons/Favorite';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import CommentIcon from '@material-ui/icons/Comment';
+import Comment from './imageBox/Comment'
+
+//#region style
+const useStyles = makeStyles(theme => ({
+	root: {
+		maxWidth: 345,
+		fontSize: '14px',
+	},
+	text: {
+		fontSize: '11px',
+	},
+	subtext: {
+		fontSize: '10px',
+	},
+	media: {
+		height: 0,
+		paddingTop: '100%', // 16:9
+	},
+	expand: {
+		transform: 'rotate(0deg)',
+		marginLeft: 'auto',
+		transition: theme.transitions.create('transform', {
+			duration: theme.transitions.duration.shortest,
+		}),
+	},
+	expandOpen: {
+		transform: 'rotate(180deg)',
+	},
+}));
+//#endregion
+
+const ImageBox = (props) => {
+	const classes = useStyles();
+	const [expanded, setExpanded] = React.useState(false);
+
+	const handleExpandClick = () => {
+		setExpanded(!expanded);
+	};
+	return (
+		<Card className={classes.root}>
+			<Typography className={classes.root}>
+				{props.image.user}
+			</Typography>
+			<Typography className={classes.subtext}>
+				{props.image.post}
+			</Typography>
```

Comment:
I don't see the point of the text here. Can you explain its purpose?

Reply:
This text is a placeholder of the repost variable that is to be implemented in the future.